packages/web-components/src/helpers/events.ts

```diff
@@ -0,0 +1 @@
+export const contextSettingsUpdated: string = 'relewise-ui-components:contextSettingsUpdated';
```

Comment:
Kan vi kører kebab all the way?

Og jeg havde nok drømt om en lille wrapper class:

```
export const Events = {
    contextSettingsUpdated: 'relewise-ui-components:context-settings-updated
};
```

Reply:
https://blog.webdevsimplified.com/2022-04/js-custom-events/

Grunden til jeg har valgt dette er fordi at der anbefales at prefix events med projektets navn og så har de navngivet alt med camel case. Jeg kan dog godt se ideen i at kære det hele med kebab. 🙂

Reply:
Jeg har implementer den ændring du efterspurgte. 🙂

Reply:
Jeg læste denne: https://gomakethings.com/custom-event-naming-conventions-in-vanilla-js/